gradle-jdks-excavator-configurations/src/main/java/com/palantir/gradle/jdks/GradleJdksConfigurator.java

```diff
@@ -110,13 +93,13 @@ private static void writeInstallationScripts(Path targetDir) {
         GradleJdksConfigsUtils.copyResourceToPath(scriptsDir, "gradle-jdks-setup.jar");
     }
 
-    private static String resolveLocalPath(JdkSpec jdkSpec) {
-        return String.format(
-                "%s-%s-%s", jdkSpec.distributionName(), jdkSpec.release().version(), jdkSpec.consistentShortHash());
+    private static String resolveLocalPath(JdkDistributionName jdkDistributionName, JdkRelease jdkRelease) {
+        return String.format("%s-%s", jdkDistributionName, jdkRelease.version());
```

Comment:
I wonder, if for absolute safety, we should include the os name and arch in the local path. These were being included in the hash before (now gone).

It might seem like this is unnecessary as the OS ~~or arch~~ does not change; but that's not necessarily the case. 

1. ~~You can run programs as if they were x86-64 in macos using the `arch` command~~ This doesn't do what I thought it did
2. On windows (if we ever support it), if you run normally once it will install windows jdks here, then if you load the project with WSL it will still use the windows versions even though you are "running linux". I don't actually know how bad this, it might actually still just work.

Reply:
Eh, on seconds thoughts this is probably fine. If it ever does become an issue, we can just add the os and/or arch and it'll work out.